procedures/katello/release.md.erb

```diff
@@ -0,0 +1,44 @@
+# When Ready to Release
+
+## Release Owner
+
+* [ ] Request Hammer CLI Katello release from maintainers
+<% unless is_rc %>* [ ] Change Redmine version <%= full_version %> state to Closed
+<% end %>
+* [ ] Clone [tool_belt](https://github.com/theforeman/tool_belt) and run:
+  * [ ] `./tools.rb setup-environment configs/foreman/<%= branch %>.yaml`
+  * [ ] `./tools.rb cherry-picks --version <%= full_version %> configs/foreman/<%= branch %>.yaml`
+  * [ ] Verify tickets in the cherry_picks_<%= full_version %> file are accounted for or additional cherry pick them
+
+* [ ] In katello release branch:
+  * [ ] Update `lib/katello/version.rb` to <%= full_version %>
+  * [ ] Commit: `git commit -m "Release <%= full_version %>"`
+  * [ ] Tag: `git tag -s -m "Release <%= full_version %>" <%= full_version %>`
+  * [ ] Push: `git push --follow-tags`
+  * [ ] Generate source gem: `gem build katello.gemspec`
+  * [ ] Push gem: `gem push katello*.gem`
```

Comment:
Could we predict the version name here? Avoids potential issues where there are other files still around